content/en/docs/concepts/sdk-configuration/otlp-exporter-configuration.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "OTLP Exporter Configuration"
+description: >-
+ Environment variables for configuring your OTLP Exporter.
+weight: 2
+---
+
+## Endpoint Configuration
+
+The following environment variables let you configure an OTLP/gRPC or OTLP/HTTP
```

Comment:
Following env vars are not documented:

- `OTEL_EXPORTER_OTLP_PROTOCOL`
- `OTEL_EXPORTER_OTLP_TRACES_PROTOCOL `
- `OTEL_EXPORTER_OTLP_METRICS_PROTOCOL `